client/my-sites/comments/comment/comment-content.jsx

```diff
@@ -18,7 +18,7 @@ import Emojify from 'components/emojify';
 import QueryComment from 'components/data/query-comment';
 import { stripHTML, decodeEntities } from 'lib/formatting';
 import getParentComment from 'state/selectors/get-parent-comment';
```

Comment:
Should this `getParentComment` selector also be moved to `state/comments/selectors` with the `state/comments/init` import added?

Reply:
Yes, it should! Excellent catch, @tyxla, totally missed this one!